### ISSUE TITLE
[$100] SRM Editorials Formatting #12

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/contentful-ops#7]
This is a follow-up to challenge https://www.topcoder.com/challenges/30122094. Please, read its spec first for details and context what needs to be done here.

**Additional notes to challenge spec:**
- The source pages are little different layout but the general purpose is the same: to extract the content and properly transfer it to Thrive article matching [this example](https://drive.google.com/file/d/1Q5SDKf4z-WIKTPtd9oMk_zGkecfOVybs/view?usp=sharing);
- Some of the source pages use MathJax. If that is the case you will need to switch its rendering to "Plain Source". You can do this with right-click on a formula and set this setting. It is a one-time config than it stays saved.
![image](https://user-images.githubusercontent.com/5585002/80075651-b72a3d80-8553-11ea-8b48-e71a58950445.png)

- If the source pages have no "problem stat tables" it is okay to skip them;
- If a problem name/title has no link to problem details page you should extract it from the [archive system](https://www.topcoder.com/tc?module=ProblemArchive) like shown on this video https://drive.google.com/file/d/1k1QkwqiKWwM-CA_Gfqkn2jcYYz5T1Eks/view. If problem is not in archive skip linking it and just create it as a heading(H3)
- In case images are present please **upload and link them** to Contentful. Then export them with your file. Uploading images to Contentful is easy via the UI:
![image](https://user-images.githubusercontent.com/5585002/80076129-5fd89d00-8554-11ea-9f9d-04f24c370553.png)

**Pages in scope:**
SRM 527 - SRM 556 from https://docs.google.com/spreadsheets/d/1GPaKO6BwJirkU5__lACbLo_6FHy-YKUVVFFoJMJffQ0/edit#gid=0&range=2:31

**Contentful file to import:** 
https://drive.google.com/file/d/1NA1YCtmKjxSLL5IeJOGeVjGqDbzihmOF/view?usp=sharing

**Comment by kkartunov** [comment]
Contest https://www.topcoder.com/challenges/30123379 has been created for this ticket.<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```

**Comment by kkartunov** [comment]
@topcoder-platform/thrive-writers @topcoder-platform/topcodercompetitors this is now open for working. Please, request an assignment here. Thank You!

**Comment by PrakashDurlabhji** [comment]
@kkartunov Kindly assign me thanks

**Comment by kkartunov** [comment]
@PrakashDurlabhji please go ahead. Check the requirements before starting!

**Comment by kkartunov** [comment]
Contest https://www.topcoder.com/challenges/30123379 has been updated - it has been assigned to PkDurlabhji.<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```

**Comment by PrakashDurlabhji** [comment]
@kkartunov sorry but u can open it for others thank you

**Comment by PrakashDurlabhji** [comment]
@iversionlv u can take it if u want

**Comment by iversonLv** [comment]
@kkartunov Can I try it?
Thanks

**Comment by kkartunov** [comment]
@iversonLv yes, please go-ahead. Thank You

**Comment by kkartunov** [comment]
Contest https://www.topcoder.com/challenges/30123379 has been updated - it has been assigned to iversonLv.<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```

**Comment by iversonLv** [comment]
@kkartunov if the SRM chars are exceeded, how should I do?
![image](https://user-images.githubusercontent.com/4952611/80393796-92cbb980-88e3-11ea-9b6d-9ca9695f2d20.png)

**Comment by iversonLv** [comment]
I think this won't need it, right?
They are useless
![image](https://user-images.githubusercontent.com/4952611/80395393-b2fc7800-88e5-11ea-808d-0dba0f3f5544.png)

**Comment by kkartunov** [comment]
@iversonLv 
- if > 50000 please split into two articles. For instance SRM XXX - Part 1 and SRM XXX - Part 2
- yes, not needed this section in red on your image. Please, do **not** add it to content.

**Comment by iversonLv** [comment]
@kkartunov

1 Can't not submit txt at https://www.topcoder.com/challenges/30122094?tab=details
2 BTW, when should I provide the export json file? Here or?

**Comment by iversonLv** [comment]
Ready for review.

**Comment by iversonLv** [comment]
Critical issue: I import the provided json, but all of them do not have creation date, I believe the date format in excel is wrong by previous google sheet scaper member? How could I export entries as DRAFT state?
![image](https://user-images.githubusercontent.com/4952611/80460705-cc91d400-8966-11ea-9992-9fed5e5e0454.png)

![image](https://user-images.githubusercontent.com/4952611/80460889-07940780-8967-11ea-93aa-cf2a97cba2e6.png)

**Comment by iversonLv** [comment]
Ok I found the command line at another ticket
--include-drafts true

**Comment by iversonLv** [comment]
https://drive.google.com/file/d/1lZJVKlRcMrl2CW8K4IKiLgG6Rh0jplQD/view?usp=sharing
this is my export file

**Comment by iversonLv** [comment]
total 38, 33articles, 3 are part2
![image](https://user-images.githubusercontent.com/4952611/80461625-22b34700-8968-11ea-8eb9-9c404b154021.png)

**Comment by iversonLv** [comment]
@kkartunov 
I do some update, please use this. Initial json file I forget to upload image to contentful space
Thanks.
https://drive.google.com/file/d/112_4JBQwwInLqvdBRhG1pAyG5-lI6xFT/view?usp=sharing

**Comment by iversonLv** [comment]
![image](https://user-images.githubusercontent.com/4952611/80609719-077d3000-8a6b-11ea-8b50-c20d25f42e81.png)

**Comment by kkartunov** [comment]
Very good work @iversonLv, Thank You!

**Comment by kkartunov** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30123379<br/><br/>```This is an automated message for kiril.kartunov via Topcoder X```
